base/file.jl

```diff
@@ -28,19 +28,34 @@ export
 
 # get and set current directory
 
+"""
+    pwd() -> AbstractString
+
+Get the current working directory.
+"""
 function pwd()
     b = Array{UInt8}(1024)
     len = Ref{Csize_t}(length(b))
     uv_error(:getcwd, ccall(:uv_cwd, Cint, (Ptr{UInt8}, Ptr{Csize_t}), b, len))
     String(b[1:len[]])
 end
 
+"""
+    cd(dir::AbstractString=homedir())
+
+Set the current working directory.
+"""
 function cd(dir::AbstractString)
     uv_error("chdir $dir", ccall(:uv_chdir, Cint, (Cstring,), dir))
 end
 cd() = cd(homedir())
 
 if is_windows()
+    @doc """
+        cd(f::Function, dir::AbstractString=homedir())
+
+    Temporarily changes the current working directory and applies function `f` before returning.
+    """ ->
```

Comment:
Probably simpler to put the docs that have OS-specific definitions after the `if` blocks using the same syntax that was being used in the helpdb. Same with the other ones further down. That would avoid having to always check whether all the duplicates get updated when changes are made to one of them.